Sources/SecureXPC/Client/XPCClient.swift

```diff
@@ -435,15 +435,13 @@ public class XPCClient {
             //   Will be delivered to the connection’s event handler if the remote service exited. The connection is
             //   still live even in this case, and resending a message will cause the service to be launched on-demand.
             //
-            // While Apple's documentation is technically correct, it's misleading in the case of an anonymous
-            // connection where there is no service. Because there is no service, there is nothing to be relaunched
-            // on-demand. The connection might technically still be alive, but resending a message will *not* work.
+            // From observed behavior Apple's documentation is *not* correct. After the connection is interrupted the
+            // subsequent call will result in XPC_ERROR_CONNECTION_INVALID and the service will not be relaunched.
```

Comment:
I think it's worth dropping a link in here to `https://github.com/trilemma-dev/SecureXPC/issues/70`, which can elaborate further.

Reply:
Will do